[2024-09-12T02:23:18.703+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-12T02:23:18.784+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_V6.second_task scheduled__2023-07-30T00:00:00+00:00 [queued]>
[2024-09-12T02:23:18.807+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_V6.second_task scheduled__2023-07-30T00:00:00+00:00 [queued]>
[2024-09-12T02:23:18.810+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-12T02:23:18.922+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): second_task> on 2023-07-30 00:00:00+00:00
[2024-09-12T02:23:18.938+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=10119) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-12T02:23:18.941+0000] {standard_task_runner.py:72} INFO - Started process 10136 to run task
[2024-09-12T02:23:18.937+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_V6', 'second_task', 'scheduled__2023-07-30T00:00:00+00:00', '--job-id', '20639', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpgu6t4caz']
[2024-09-12T02:23:18.952+0000] {standard_task_runner.py:105} INFO - Job 20639: Subtask second_task
[2024-09-12T02:23:19.165+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_V6.second_task scheduled__2023-07-30T00:00:00+00:00 [running]> on host 5508ea9ebfaf
[2024-09-12T02:23:19.427+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='acap' AIRFLOW_CTX_DAG_ID='our_first_dag_V6' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2023-07-30T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-07-30T00:00:00+00:00'
[2024-09-12T02:23:19.432+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-12T02:23:19.490+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-12T02:23:19.493+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, this is the second task!']
[2024-09-12T02:23:19.542+0000] {subprocess.py:86} INFO - Output:
[2024-09-12T02:23:19.547+0000] {subprocess.py:93} INFO - hey, this is the second task!
[2024-09-12T02:23:19.548+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-12T02:23:19.631+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-12T02:23:19.633+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_V6, task_id=second_task, run_id=scheduled__2023-07-30T00:00:00+00:00, execution_date=20230730T000000, start_date=20240912T022318, end_date=20240912T022319
[2024-09-12T02:23:19.827+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-12T02:23:19.866+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
